signer/src/block_observer.rs

```diff
@@ -440,7 +642,7 @@ mod tests {
     }
 
     impl EmilyInteract for () {
-        async fn get_deposits(&mut self) -> Vec<DepositRequest> {
+        async fn get_deposits(&mut self) -> Vec<CreateDepositRequest> {
```

Comment:
This should likely be swapped for some autogenerated struct one day. Just flagging, for now it's fine and we may ultimately need to ship with this depending on timing.

Reply:
Yeah, I figured we'll need to switch types eventually.